purchase_line_procurement_group/views/purchase.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="purchase_order_form_inherit" model="ir.ui.view">
```

Comment:
aesthetic change (optional):
use original id (purchase_order_form)

Reply:
Fixed. Thank you.